modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/field_validation_support.js

```diff
@@ -40,20 +40,22 @@ AUI.add(
 				if (!instance.get('readOnly')) {
 					var evaluator = instance.get('evaluator');
 
-					evaluator.evaluate(
-						instance,
-						function(result) {
-							if (callback) {
-								var hasErrors = instance.hasErrors();
-
-								if (!result || !Lang.isObject(result)) {
-									hasErrors = true;
+					if (evaluator) {
+						evaluator.evaluate(
+							instance,
+							function(result) {
+								if (callback) {
```

Comment:
To avoid callback hell change this implementation for `if(!callback) { return; }`

Reply:
nice!